rust/tests/tests/test_meta_string.rs

```diff
@@ -0,0 +1,244 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use std::iter;
+
+use fury::{Encoding, MetaStringDecoder, MetaStringEncoder};
+
+#[test]
+fn test_encode_meta_string_lower_special() {
+    let encoder = MetaStringEncoder::new();
+    let bytes1 = encoder.encode_lower_special("abc_def").unwrap();
+    assert_eq!(bytes1.len(), 5);
+    let bytes2 = encoder
+        .encode("org.apache.fury.benchmark.data")
+        .unwrap()
+        .bytes;
+    assert_eq!(bytes2.len(), 19);
+    let bytes3 = encoder.encode("MediaContent").unwrap().bytes;
+    assert_eq!(bytes3.len(), 9);
+    // 验证解码
```

Comment:
Could you use English for all comments and documents?

Reply:
sorry about that